internal/provider/resource_gitlab_project_level_mr_approvals_test.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 func TestAccGitlabProjectLevelMRApprovals_basic(t *testing.T) {
+	testAccCheckEE(t)
```

Comment:
@timofurrer  - very much a nitpick, but since we're not creating any resources via the API before `resource.Test`, it feels like this should be moved into the PreCheck function instead. WDYT?

Reply:
Yes, you are right. I've refactored the test into creating the project out of the tf config :)